app/components/WoW/DateRangeControls.tsx

```
@@ -0,0 +1,79 @@
+  allTimestamps,
+  onStartDateChange,
+  onEndDateChange,
+  formatTimestamp
+}: DateRangeControlsProps) {
+  return (
+    <div className="bg-white dark:bg-gray-800 p-4 rounded-lg shadow">
+      <div className="grid grid-cols-1 md:grid-cols-3 gap-4">
+        <div>
+          <label className="block text-sm font-medium text-gray-900 dark:text-gray-100 mb-1">
+            Start Date
+          </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)